charmdir_test.go

```diff
@@ -110,7 +155,7 @@ func (s *CharmDirSuite) TestArchiveToWithSymlinkedRootDir(c *gc.C) {
 	s.assertArchiveTo(c, baseDir, charmDir)
 }
 
-func (s *CharmDirSuite) assertArchiveTo(c *gc.C, baseDir, charmDir string) {
+func (s *CharmDirSuite) assertArchiveTo(c *gc.C, baseDir string, charmDir string) {
```

Comment:
no need for this change, it can be reverted

Reply:
done